core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -66,6 +66,9 @@ public interface CalciteResource {
   @BaseMessage("Geo-spatial extensions and the GEOMETRY data type are not enabled")
   ExInst<SqlValidatorException> geometryDisabled();
 
+  @BaseMessage("Proj4J EPSG is missing from the classpath")
+  ExInst<CalciteException> proj4jEpsgIsMissing();
```

Comment:
I would add '; to resolve this problem, download the EPSG data set and agree to its terms of use'